html.go

```diff
@@ -482,8 +482,8 @@ func (r *HTMLRenderer) outHRTag(w io.Writer) {
 }
 
 // RenderNode is a default renderer of a single node of a syntax tree. For
-// block nodes it will be called twice: first time with entering=true, second
-// time with entering=false, so that it could know when it's working on an open
+// block nodes it will be called twice: first time with entering=True, second
+// time with entering=False, so that it could know when it's working on an open
```

Comment:
Wow, that's a brain fart. Obviously too much Python lately :-). I'll fix it, thanks!